packages/store-ui/src/ProductQuantity/NumericStepper.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC, Fragment, useCallback } from 'react'
+import { Button, Flex, Input } from 'theme-ui'
+
+export interface NumericStepperProps {
+  variant?: string
+  minValue: number | 1
+  maxValue?: number | undefined
+}
+
+var currentQuantity = 1
+
+export const getCurrentQuantity = () => {
+  return currentQuantity
+}
+
+export const NumericStepper: FC<NumericStepperProps> = ({
+  variant = 'productQuantity',
+  minValue,
+  maxValue,
+  children,
+}) => {
+  var [quantity, setQuantity] = React.useState<number>(1)
+
+  const updateQuantity = (value: number) => {
+    currentQuantity = value
```

Comment:
instead of saving the value of the quantity in a global and local variable, we can just save locally as you already do with useState and return a handle using the onChange prop so that every time there is a change in the quantity number, this prop would pass out the new value.